packages/whenable/src/module.js

```diff
@@ -0,0 +1,27 @@
+/* global globalThis */
+// @ts-check
+import { makeWhen } from './when.js';
+import { prepareWhenableKits } from './whenable.js';
+import { prepareWatch } from './watch.js';
+import makeE from './E.js';
+
+/**
+ * @param {import('@agoric/base-zone').Zone} zone
+ * @param {object} [powers]
+ * @param {(reason: any) => boolean} [powers.rejectionMeansRetry]
+ * @param {(p: PromiseLike<any>, watcher: import('./watch.js').PromiseWatcher, ...args: unknown[]) => void} [powers.watchPromise]
+ */
+export const prepareWhenableModule = (zone, powers) => {
```

Comment:
I understand why "module" seems like a good term here. But given our ever heavier involvement with JS module systems and proposals, I think it would be good to choose another term here.

Normally I'd suggest prepareSomethingKit. But I don't have a concrete suggestion. prepareWhenableKit would be confusing because it is not at all at the same level as makeWhenableKit.

Reply:
Yes, jives, thanks!

Reply:
I see already fixed. Resolving.